docs/commands/zq.md

```diff
@@ -1,6 +1,7 @@
 ---
 sidebar_position: 0
 sidebar_label: zq
+description: A command-line tool that uses the Zed Language for pipeline-style search and analytics.
```

Comment:
Did you mean to add a description to just this file?

Reply:
Yes, since the first line of the content starts with "tldr". Doesn't seem too classy to include that in a twitter card link.